views/templates/admin/tabs/faviconConfiguration.tpl

```diff
@@ -0,0 +1,103 @@
+{*
+* 2007-2018 PrestaShop
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author    PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2018 PrestaShop SA
+* @license   http://addons.prestashop.com/en/content/12-terms-and-conditions-of-use
+* International Registered Trademark & Property of PrestaShop SA
+*}
+<div class="panel col-lg-10 right-panel">
+    <form method="post" action="{$moduleAdminLink|escape:'htmlall':'UTF-8'}&page=faviconConfiguration" class="form-horizontal">
+        <h3>
+            {l s='Order Notifications on the Favicon Configuration' d='Modules.Faviconnotificationbo.Admin'}
+        </h3>
+        <div class="form-group row">
+            <div class="title">
+                {l s='Show notification on browser tab for' d='Modules.Faviconnotificationbo.Admin'}
+            </div>
+            <ol>
+                <div class="col-lg-5 col-md-4 col-xs-10">
+                    <div class="form-group">
+                        <div class="control-label col-lg-5 col-md-4 col-xs-10">
+                            <label class="labelbutton">{l s='New Orders' d='Modules.Faviconnotificationbo.Admin'}</label>
+                        </div>
+                        <div>
+                            <div class="input-group fixed-width-lg">
+                                <span class="switch prestashop-switch fixed-width-lg">
+                                <input class="yes" type="radio" name="CHECKBOX_ORDER" id="checkbox_track_new_orders_on" value="1" {if $bofavicon_params.CHECKBOX_ORDER eq 1}checked="checked"{/if}>
+                                <label for="checkbox_track_new_orders_on" class="radioCheck">{l s='YES' d='Modules.Faviconnotificationbo.Admin'}</label>
```

Comment:
Does it need to be in upper cases? Otherwise, you can just write _Yes_ and localize it in _Admin.Global_.